.yarn/patches/app-builder-lib-npm-24.4.0-05322ff057.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/templates/nsis/include/allowOnlyOneInstallerInstance.nsh b/templates/nsis/include/allowOnlyOneInstallerInstance.nsh
+index a1fd1875d852ff69c087a3103eff827c20d37ca5..5222614ddad3276876857e7a9dde4017a6b9fc85 100644
+--- a/templates/nsis/include/allowOnlyOneInstallerInstance.nsh
++++ b/templates/nsis/include/allowOnlyOneInstallerInstance.nsh
+@@ -42,7 +42,7 @@
+     ${nsProcess::FindProcess} "${_FILE}" ${_ERR}
+   !else
+     # find process owned by current user
+-    nsExec::Exec `cmd /c tasklist /FI "USERNAME eq %USERNAME%" /FI "IMAGENAME eq ${_FILE}" | %SYSTEMROOT%\System32\find.exe "${_FILE}"`
++    nsExec::Exec `cmd /c tasklist /FI "USERNAME eq %USERNAME%" /FI "PID ne $pid" /FI "IMAGENAME eq ${_FILE}" | %SYSTEMROOT%\System32\find.exe "${_FILE}"`
```

Comment:
See the explanation for this change here: https://github.com/electron-userland/electron-builder/issues/6865#issuecomment-1630384113

This seems to fix #8149. To test it, I
1. Installed a copy of Joplin with the updated installer and opened Joplin.
2. Opened a new copy of the installer while Joplin was running and had the installer close the exiting Joplin application (then proceeded with the re-installation).
3. Re-installed Joplin using the installer with no running copies of Joplin.

All tests used the "only for me" option in the dialog.